feature-libs/product-configurator/rulebased/components/attribute/types/radio-button/configurator-attribute-radio-button.component.spec.ts

```diff
@@ -182,6 +182,38 @@ describe('ConfigAttributeRadioButtonComponent', () => {
       );
     });
 
+    it('should display quantity and no price', () => {
+      component.attribute.quantity = 5;
+      component.attribute.attributePriceTotal = {
+        currencyIso: '$',
+        formattedValue: '500.00$',
+        value: 500,
+      };
+
+      let value = component.attribute.values
+        ? component.attribute.values[0]
+        : undefined;
+      if (value) {
+        value.valuePrice = undefined;
+      } else {
+        fail('Value not available');
+      }
+
+      fixture.detectChanges();
```

Comment:
Line #186 until #202 are not needed because value does not carry a value price

Reply:
Better you create an additional test that adds a value price. In this case, the UI rendering should be different

Reply:
This test already exists: `should display quantity and price` line 212